addons/binding/pom.xml

```diff
@@ -87,6 +87,7 @@
     <module>org.openhab.binding.systeminfo.test</module>
     <module>org.openhab.binding.tankerkoenig</module>
     <module>org.openhab.binding.tellstick</module>
+    <module>org.openhab.binding.te923</module>
```

Comment:
it should be above `tellstick` While it looks like a number 1 it's actually an character L.
Also you need to add your binding to the `feature.xml`